README.md

```diff
@@ -72,24 +71,27 @@ Replace value of `image` field with your catalog source image.
 
 6. Deploy Che using chectl:
 ```sh
-chectl server:start --installer=olm --multiuser --platform=openshift -n che-namespace --catalog-source-yaml /home/user/path/to/custom-catalog-source.yaml --olm-channel=nightly --package-manifest-name=eclipse-che-preview-openshift
+chectl server:start --installer=olm --multiuser --platform=openshift \
+  -n che-namespace \
+  --catalog-source-yaml /home/user/path/to/custom-catalog-source.yaml \
+  --package-manifest-name=eclipse-che-preview-openshift \
+  --olm-channel=nightly 
 ```
 
 ### OpenShift oAuth
 
-Bear in mind that che-operator service account needs to have cluster admin privileges so that the operator can create oauthclient at a cluster scope.
-There is `oc adm` command in both deploy scripts. Uncomment it if you need this feature.
-Make sure your current user has cluster-admin privileges.
+Che-operator service account needs to have cluster admin privileges so that the operator can create a cluster-wide oauthclient. 
+
+Log in using a user with cluster admin privileges, then uncomment the `oc adm` commands in both deploy scripts to enable this feature.
 
 ### TLS
 
-TLS is enabled by default.
-Turning it off is not recommended as it will cause malfunction of some components.
+TLS is enabled by default. Disabling TLS is not supported as it will cause webviews used by the default IDE, Eclipse Theia, to misbehave. Note that while Che needs to use HTTPS for its routes, an insecure HTTP proxy is still possible.
```

Comment:
vague statements like "some components" won't discourage people. Specific reasons will. Also wanted to drive home the fact that while https is needed in routes, we can still have an http proxy config. Correct?